schema/course.schema.json

```diff
@@ -33,8 +33,7 @@
                       "type": "number",
                       "title": "Close button Nav order",
                       "description": "Determines the order in which the object is displayed in the navigation bar. Negative numbers (e.g. -100) are left-aligned. Positive numbers (e.g. 100) are right-aligned.",
-                      "default": 150,
-                      "validators": []
+                      "default": 150
```

Comment:
@joe-allen-89 Instead of removing the empty validator, should we validate this as a number?

"validators": ["number"]

Reply:
@swashbuck none of the new style schemas have numbers validators, I think we'll be ok to drop this.